src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -1,23 +1,70 @@
 package edu.kis.powp.jobs2d.drivers;
 
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-
 import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.features.DriverFeature;
 
-public class SelectDriverMenuOptionListener implements ActionListener {
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Observable;
+import java.util.Observer;
+
+public class SelectDriverMenuOptionListener extends Observable implements ActionListener {
 	private DriverManager driverManager;
 	private Job2dDriver driver = null;
+	public static SelectDriverMenuOptionListener instance;
+	DriverFeature df;
 
 	public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
 		this.driverManager = driverManager;
 		this.driver = driver;
+		this.instance = this;
+		this.df = new DriverFeature();
```

Comment:
There is no need to create instances of DriverFeature (only static fields and methods)